internal/charmstore/addentity.go

```diff
@@ -412,7 +412,7 @@ func (s *Store) addCharm(c charm.Charm, p addParams) (err error) {
 	// final gateway before a potentially invalid url might be stored
 	// in the database.
 	id := p.url.URL
-	logger.Infof("add charm url %s; prev %d; dev %v", &id, p.url.PromulgatedRevision, p.url.Development)
+	logger.Infof("add charm url %s; prev %d", &id, p.url.PromulgatedRevision)
```

Comment:
s/prev/promulgated rev/ ? as I always read that as "previous" and it's confusing [shrug]